Archipelago.HollowKnight/ArchipelagoRandomizer.cs

```diff
@@ -301,15 +309,17 @@ public void PlaceItem(string location, string name, NetworkItem netItem)
                             costs.Add(new ItemChanger.Modules.CumulativeRancidEggCost(entry.Value));
                             break;
                         default:
-                            Archipelago.Instance.LogError($"Encountered UNKNOWN currency type {entry.Key} at location {originalLocation}!");
+                            Archipelago.Instance.LogWarn($"Encountered UNKNOWN currency type {entry.Key} at location {originalLocation}!");
```

Comment:
I want to do an entire pass on what level all of our logging is eventually, this is a good reminder.